apps/communications/contacts/js/views/details.js

```diff
@@ -687,6 +706,8 @@ contacts.Details = (function() {
     'setContact': setContact,
     'toggleFavorite': toggleFavorite,
     'render': render,
-    'defaultTelType' : DEFAULT_TEL_TYPE
+    'defaultTelType': DEFAULT_TEL_TYPE,
+    'startNFC': startNFC,
+    'stopNFC': stopNFC
```

Comment:
not used, but for consistency decided on making it accesible. Not a problem if we make it private.